7.4.26-fpm-alpine3.15/docker-entrypoint.sh

```diff
@@ -10,6 +10,7 @@ export MEMCACHED_ENABLE=${MEMCACHED_ENABLE:-0}
 export REDIS_ENABLE=${REDIS_ENABLE:-0}
 export XDEBUG_ENABLE=${XDEBUG_ENABLE:-0}
 export LDAP_ENABLE=${LDAP_ENABLE:-0}
+export APCU_ENABLE=${APCU_ENABLE:-1}
```

Comment:
You put this line only in this entrypoint. I think we should have the default definition also in all the others.

Reply:
nice catch, i missed that!

Reply:
Added